common/path.cc

```diff
@@ -974,9 +974,8 @@ absl::Status CreateSymlink(const std::string& target,
                                               error_code);
   }
   if (error_code) {
```

Comment:
If you address my comments above about `ErrorCodeToCanonicalStatus`, then you can simplify this code to:

```
return ErrorCodeToCanonicalStatus(
        error_code,
        "Failed to create symlink '%s' with target '%s'",
        link_path, target);
```

Reply:
Done.